docs/guides/LIF_Add_Data_Source.md

```diff
@@ -0,0 +1,66 @@
+# Add a New Data Source
+
+Data sources are used by the **Orchestrator** to fulfil LIF queries. These sources can be open or require authN/authZ, and return data in a variety of formats. Data sources are configured through an adapter. Reference implementations for 2 adapter flows are provided in the repository:
+- LIF to LIF
+- Example Data Source to LIF
+
+To configure a non-LIF data source that requires custom authentication or custom data access (such as pagination) using the general docker compose file, the following guide is offered. This guide will show how to:
+- Create a new adapter called `sis-data-source-to-lif` that supports a configured `Bearer` token for auth
+- Configure the **Orchestrator** with a data source from `org1` called `acme-sis-data-source` using the new adapter
+- Setup a translation via the **MDR** to map the data source values into the LIF schema
+- Confirm the data source can be queried through the **LIF API**
+
+The `example-data-source-rest-api-to-lif` adapter is the reference guide for this scenario. The flow offers flexibility in how the data source is leveraged and how generalized adopters want the _adapter_ to be for the various _data sources_.
+
+1. Ensure the data source API is reachable by code running in the `dagster-code-location` container.
+
+2. Clone `components/lif/data_source_adapters/example_data_source_rest_api_to_lif_adapter` into a sibling directory called `components/lif/data_source_adapters/sis_data_source_to_lif_adapter`.
+
+3. Adjust the code in `sis_data_source_to_lif_adapter/adapter.py` to access the source API, including:
+    - The auth token header to create a header `Authorization: Bearer [[TOKEN]]`
+    - The context path for requesting data about a specific user (use the `self.lif_query_plan_part.person_id.identifier` for the user's identifier)
+
+4. In the docker compose file for `dagster-code-location`, add the environment variables:
+```
+ADAPTERS__SIS_DATA_SOURCE_TO_LIF__ORG1_ACME_SIS_DATA_SOURCE__CREDENTIALS__HOST
+ADAPTERS__SIS_DATA_SOURCE_TO_LIF__ORG1_ACME_SIS_DATA_SOURCE__CREDENTIALS__SCHEME
+ADAPTERS__SIS_DATA_SOURCE_TO_LIF__ORG1_ACME_SIS_DATA_SOURCE__CREDENTIALS__TOKEN
+```
+
```

Comment:
At the moment, these instructions assume that you are already happy with your OrgLIF data model. There is a decent chance that when people start working on their translations, they'll notice that the OrgLIF data model is missing fields that they need. In that case, they should first review the Base LIF data model to see if it already exists and just needs to be marked as included in their Org LIF model. Then if it does not exist, they should use the dot buttons to create the Entities and Attributes that they need in their Org LIF model (please do not modify Base LIF). As of today, value sets created in the Data Models tab cannot be taken into consideration when doing translations. If someone needs to map a value set, they will need to handle that manually in the JSONata instruction.

Reply:
I'm going to leave out the bit about 'If they want their changes to Org LIF committed to a Github branch, then they will need to go through the typical process for this that also updates the SQL files.'. #17 and the follow up import/export of mappings should resolve the need to make SQL file changes for data model changes.